### PR TITLE
docs: fix pr base path

### PR DESCRIPTION
**Pull request by kayman233**
Прокинул в сам basePath дополнительный путь для главной страницы документации, чтобы корректно забирались файлы в pr версиях доки

:baby_chick: Download canary assets:

[colors--canary.439.4510440418.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.439.4510440418.xml)  
[color_metro_ios-swift--canary.439.4510440418.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.439.4510440418.swift)  
[color_metro_kotlin--canary.439.4510440418.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.439.4510440418.kt)  
[color_metro_react-native--canary.439.4510440418.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.439.4510440418.ts)  
[color_sberHealth_ios-swift--canary.439.4510440418.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.439.4510440418.swift)  
[color_sberHealth_kotlin--canary.439.4510440418.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.439.4510440418.kt)  
[color_sberHealth_react-native--canary.439.4510440418.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.439.4510440418.ts)  
[color_sbermarket_ios-swift--canary.439.4510440418.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.439.4510440418.swift)  
[color_sbermarket_kotlin--canary.439.4510440418.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.439.4510440418.kt)  
[color_sbermarket_react-native--canary.439.4510440418.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.439.4510440418.ts)  
[color_sberprime_ios-swift--canary.439.4510440418.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.439.4510440418.swift)  
[color_sberprime_kotlin--canary.439.4510440418.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.439.4510440418.kt)  
[color_sberprime_react-native--canary.439.4510440418.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.439.4510440418.ts)  
[color_selgros_ios-swift--canary.439.4510440418.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.439.4510440418.swift)  
[color_selgros_kotlin--canary.439.4510440418.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.439.4510440418.kt)  
[color_selgros_react-native--canary.439.4510440418.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.439.4510440418.ts)  
[color_smbusiness_ios-swift--canary.439.4510440418.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.439.4510440418.swift)  
[color_smbusiness_kotlin--canary.439.4510440418.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.439.4510440418.kt)  
[color_smbusiness_react-native--canary.439.4510440418.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.439.4510440418.ts)  
[PlasmaTokensColor--canary.439.4510440418.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.439.4510440418.swift)  
[shadow_sbermarket_react-native--canary.439.4510440418.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.439.4510440418.ts)  
[typo_mage_ios-swift--canary.439.4510440418.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.439.4510440418.swift)  
[typo_mage_kotlin--canary.439.4510440418.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.439.4510440418.kt)  
[typo_mage_react-native--canary.439.4510440418.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.439.4510440418.ts)  
[typo_plasma_ios-swift--canary.439.4510440418.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.439.4510440418.swift)  
[typo_plasma_kotlin--canary.439.4510440418.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.439.4510440418.kt)  
[typo_plasma_react-native--canary.439.4510440418.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.439.4510440418.ts)  
[typo_ruler_ios-swift--canary.439.4510440418.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.439.4510440418.swift)  
[typo_ruler_kotlin--canary.439.4510440418.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.439.4510440418.kt)  
[typo_ruler_react-native--canary.439.4510440418.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.439.4510440418.ts)  
[typo_sage_ios-swift--canary.439.4510440418.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.439.4510440418.swift)  
[typo_sage_kotlin--canary.439.4510440418.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.439.4510440418.kt)  
[typo_sage_react-native--canary.439.4510440418.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.439.4510440418.ts)  
[typo_sbermarket_ios-swift--canary.439.4510440418.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.439.4510440418.swift)  
[typo_sbermarket_kotlin--canary.439.4510440418.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.439.4510440418.kt)  
[typo_sbermarket_react-native--canary.439.4510440418.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.439.4510440418.ts)  
[typo_soulmate_ios-swift--canary.439.4510440418.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.439.4510440418.swift)  
[typo_soulmate_kotlin--canary.439.4510440418.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.439.4510440418.kt)  
[typo_soulmate_react-native--canary.439.4510440418.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.439.4510440418.ts)

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-439/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-439/

**Comment by Salute-Eva** [comment]
Documentation preview deployed
  
  website: http://plasma.sberdevices.ru/pr/pr-439/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-439/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-439/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-439/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-439/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed
  
  website: http://plasma.sberdevices.ru/pr/pr-439/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-439/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-439/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-439/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-439/temple-storybook/

File: website/plasma-website/next.config.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const path = require('path');
 
 const { PR_NAME } = process.env;
 
-const basePath = PR_NAME ? `/${PR_NAME}` : '';
+const basePath = PR_NAME ? `/pr/${PR_NAME}` : '';
```
Comment (Yeti-or) [inline website/plasma-website/next.config.js:6]: Найс

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-439/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-439/

**Review by neretin-trike — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-439/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-439/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-439/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-439/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-439/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Review by Yeti-or — APPROVED** [review]

**Pull request merged**